sbol/__init__.py

```diff
@@ -25,7 +25,8 @@
 from .sbolerror import SBOLError
 from .sequence import Sequence
 from .sequenceannotation import SequenceAnnotation
-
+from .identified import Identified
+from .property import *
```

Comment:
No, no, a thousand times no. No wildcard imports please.

What code needs symbols defined in property.py? What symbols should be exported and maintained?

This is just plain lazy. No two ways about it.

Reply:
Fixed